contracts/bonding/BondingManager.sol

```diff
@@ -282,7 +292,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
             }
         }
 
-        return true;
+        Bond(_to, msg.sender, currentRound);
```

Comment:
Are we including `currentRound` in all the events for convenience and searchability on the frontend? Presumably the client would know which block or round it occured in based upon the event metadata itself, right...but this would just allow us to filter or easily view a timeline?

Reply:
Yeah good point - the current round can always be computed from the block number during which an event is received